command/agent/command.go

```diff
@@ -120,8 +120,12 @@ func (c *Command) readConfig() *Config {
 		return nil
 	}), "consul-client-auto-join", "")
 	flags.StringVar(&cmdConfig.Consul.ClientServiceName, "consul-client-service-name", "", "")
+	flags.StringVar(&cmdConfig.Consul.ClientHTTPHealthCheckName, "consul-client-http-health-check-name", "", "")
```

Comment:
Should the command line and configuration parameters use `check` instead of `health-check` to save a little repetition?

I'm fine with it either way, but since we're stuck with it ~forever once released if you want to shorten it to `check` please do so now. :)

Reply:
@schmichael it had crossed my mind that the flags and vars where fairly long; I will push a change to remove the `health-` from `health-check` shortly.